content/sensu-go/6.0/observability-pipeline/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Sensu Observability Pipeline"
+linkTitle: "Observability Pipeline"
+description: "Sensu's observability pipeline is a flexible, automated tool that gives you visibility into every part of your organization's infrastructure. Read this overview to learn how the observability pipeline works."
+product: "Sensu Go"
+version: "6.0"
+weight: 10
+layout: "single"
+toc: false
+menu:
+  sensu-go-6.0:
+    identifier: observability-pipeline
+---
+
```

Comment:
@cwjohnston here is the observability pipeline text that you mentioned you had some thoughts about.